Tests/Utils/XCTestCase+Extensions.swift

```diff
@@ -22,7 +23,11 @@ extension XCTestCase {
         self.waitForExpectations(timeout: self.defaultTimeout, handler: nil)
     }
 
-    func expectation(function: String = #function, name: String = "") -> XCTestExpectation {
+    func expectation(name: String, function: String = #function) -> XCTestExpectation {
```

Comment:
Continuing from above, can we revert this change?

Reply:
Done.